python/syntax.lark

```diff
@@ -159,9 +189,12 @@ value: NUMBER
      | listval
      | recordval
      | impn
+     | HEX
+     | BINARY
+     | OCTAL
 
 // Make sure that keywords are not in NAME
-NAME: /(?!(\b(let|type|var|alias|for|in|import|imp|return|if|else|class|pub|break|continue|with|yield|do|not)\b))/ CNAME
+NAME: /(?!(\b(let|type|var|alias|for|in|import|imp|return|if|else|class|pub|break|continue|with|yield|do|not|match)\b))/ CNAME
```

Comment:
Might this break backwards compatibility? or meh

Because I don't think it's an issue if `match` is not a reserved keyword (though it's interesting we didn't reserve it?)

Reply:
This was a bug, we were supposed to reserve it but it ended up being that I forgot to add `match`